ruby/ql/test/library-tests/dataflow/local/TaintStep.expected

```diff
@@ -1,9 +1,6 @@
 | file://:0:0:0:0 | [summary] read: argument position 0.any element in Hash[] | file://:0:0:0:0 | [summary] read: argument position 0.any element.element 1 or unknown in Hash[] |
-| file://:0:0:0:0 | parameter any of ;Pathname;Method[join] | file://:0:0:0:0 | [summary] to write: return (return) in ;Pathname;Method[join] |
```

Comment:
Why did all these lines disappear?

Reply:
Sorry I should have mentioned that. This test case doesn't reference `Pathname` at all, so the summaries with `Pathname` in the type column get pruned.

Pruning was previously based entirely on the package column, and was disabled for Ruby since the package column couldn't be used for anything.